winml/lib/Api/LearningModelSessionOptions.cpp

```diff
@@ -42,4 +42,14 @@ STDMETHODIMP LearningModelSessionOptions::SetIntraOpNumThreadsOverride(uint32_t
   telemetry_helper.SetIntraOpNumThreadsOverride(intraOpNumThreads);
   return S_OK;
 }
+
+bool LearningModelSessionOptions::GetIntraOpThreadSpinning() {
+  return allow_thread_spinning_;
```

Comment:
No telemetry for the getter?

Reply:
its not exposed in our idl, so its only ever used internally. Its the same pattern for GetIntraOpNumThreads.